src/components/books.css

```diff
@@ -0,0 +1,230 @@
+.book-container {
+  width: auto;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  border: 2px solid rgb(109, 103, 103);
+  box-shadow: 2px 2px 12px rgb(109, 107, 107);
+  margin: 22px auto;
+  padding: 16px;
+  border-radius: 6px;
+  align-content: center;
+  flex-wrap: wrap;
+}
+
+.book-header {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: auto;
+  margin: auto;
+}
+
+.book-category {
+  opacity: 0.5;
+  font-size: 19px;
+  font-weight: 600;
+  font-stretch: normal;
+  font-style: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.book-title {
+  font-size: 17px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+}
+
+.book-author {
+  font-size: 17px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  letter-spacing: normal;
+  color: rgb(79, 142, 197);
+}
+
+.book-actions {
+  font-size: large;
+}
+
+.book-actions button {
+  border: none;
+  font-size: 19px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  color: rgb(79, 142, 197);
+  margin: auto;
+}
+
+.book-progress {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-left: auto;
+  justify-content: space-between;
+  font-size: 22px;
+}
```

Comment:
- [ ] Well done styling the App however your design should match the Zeplin design. Kindly change the colors, spacing and fonts to match the design.

Your design:

<img width="1287" alt="Screenshot 2023-10-06 at 6 16 43 PM" src="https://github.com/richie1988/My_Bookstore_App/assets/53347966/2c767f30-890e-49c9-9450-6bb536403eaa">


Zeplin design:


<img width="1133" alt="Screenshot 2023-10-06 at 6 18 10 PM" src="https://github.com/richie1988/My_Bookstore_App/assets/53347966/63c1d3e1-7ea1-4211-8f58-f5376a10525b">